### PR TITLE
add getBucketsByIndexes API and ioctl commands to get vote bucket data

### PR DESCRIPTION
**Pull request by yutongp**
based on #2170

### CODE REVIEW

File: ioctl/cmd/bc/bucket.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+)
+
+// bcBucketCmd represents the bc Bucket command
+var bcBucketCmd = &cobra.Command{
+	Use:   config.TranslateInLang(bcBucketUses, config.UILanguage),
+	Short: config.TranslateInLang(bcBucketCmdShorts, config.UILanguage),
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := getBucket(args[0])
```
Comment (dustinxie) [inline ioctl/cmd/bc/bucket.go:51]: we'll need to parse out a list of index from args[i]
like 47 56 77 86 100 372 7328 72727

File: ioctl/cmd/bc/bucket.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+		config.English: "bucket [BUCKET_INDEX]",
+		config.Chinese: "bucket [票索引]",
+	}
+)
+
+// bcBucketCmd represents the bc Bucket command
+var bcBucketCmd = &cobra.Command{
+	Use:   config.TranslateInLang(bcBucketUses, config.UILanguage),
+	Short: config.TranslateInLang(bcBucketCmdShorts, config.UILanguage),
+	Args:  cobra.ExactArgs(1),
```
Comment (dustinxie) [inline ioctl/cmd/bc/bucket.go:48]: should be a range (1, inf)

File: ioctl/cmd/bc/bucket.go [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+	}
+	message := bucketMessage{
+		Node:   config.ReadConfig.Endpoint,
+		Bucket: bucket,
+	}
+	fmt.Println(message.String())
+	return nil
+}
+
+func getBucketByIndex(index uint64) (*iotextypes.VoteBucket, error) {
```
Comment (dustinxie) [inline ioctl/cmd/bc/bucket.go:142]: should take []uint64 (parsed from args[]) as input

File: ioctl/cmd/bc/bucket.go (hunk 1 above)
Comment (yutongp) [reply ioctl/cmd/bc/bucket.go:51]: i am not going to change ioctl for now. it is a client side changes which is easier if we need to this. API is harder to change later on, so it is better to change it now

**Pull request merged**